### PR TITLE
[GH-21] Migrate to DynamoDB

### LINKED ISSUE #21: Migrate to DynamoDB or other storage method
Move away from using static JSON files as a "database"

Possibly investigate other methods of datastore

### PR DESCRIPTION
**Pull request by mommothazaz123**
Resolves #21 
This change removes the old static JSON files that used to act as Taine's DB, and uses DynamoDB instead. The resource is initialized in `db.py` and imported where needed.

### CODE REVIEW

File: lib/db.py [hunk 1]
```diff
@@ -0,0 +1,112 @@
+import boto3
+
+# for use imported elsewhere
+dynamo = boto3.resource('dynamodb', endpoint_url="http://localhost:8000")
```
Comment (zmckinnon) [inline lib/db.py]: This should probably be an environment variable.

**Comment by zmckinnon** [comment]
It is probably noteworthy to note that there is a dynamodb docker image.  You should be able to run docker compose and get this all running in one command.

See: https://hub.docker.com/r/amazon/dynamodb-local/

File: lib/db.py (hunk 1 above)
Comment (mommothazaz123) [reply lib/db.py]: Done in 441988a

**Comment by madcowfred** [comment]
> It is probably noteworthy to note that there is a dynamodb docker image. You should be able to run docker compose and get this all running in one command.
> 
> See: https://hub.docker.com/r/amazon/dynamodb-local/

Not sure how useful this would be - we have no live reload (`dotnet watch`, Django `runserver`, etc) so any code changes would require restarting the containers anyway. Can probably update the docs with some quick info on pulling/starting a DynamoDB Docker image?

**Pull request merged**